### PR TITLE
enable DRVSTR for output pin

### LINKED ISSUE #311: Built-in neopixel on QT Py M0 (and probably feather M4 CAN) shows incorrect colors
On QTPy M0 boards running Arduino code, the onboard neopixel can show colors that are "red shifted" (actually, lacking in blue) from what they ought to be.
This appears to be because the neopixel on those boards is powered by a SAMD output pin, and by default that pin leaves the neopixel under-powered.

Workaround: add
```
      PORT->Group[0].PINCFG[15].bit.DRVSTR = 1;  // turn up neopixel power
```
to setup() in your sketch.  (for QTPy)

The Feather M4 CAN board seems to have the same "powered by a pin" scheme, and so it probably has the same bug.

I have some concern that at maximum brightness levels, the power consumption of the neopixel may exceed even the "high current" drive of a SAMD output pin.

https://forums.adafruit.com/viewtopic.php?f=47&t=186027

(Hmm.  This may also be a problem no the RP2040 boards with IO-powered neopixels; I think they have a similar "drive strength" setting.)

### PR DESCRIPTION
**Pull request by hathach**
- enable DRVSTR pinMode(output)
- add neopixel and button pin define for circuitplay

superseded and close #312, fix #311

### CODE REVIEW

File: cores/arduino/wiring_digital.c [hunk 1]
```diff
@@ -39,39 +39,39 @@ void pinMode( uint32_t ulPin, uint32_t ulMode )
...
       // Enable pull level (cf '22.6.3.2 Input Configuration' and '22.8.6 Data Output Value Clear')
-      PORT->Group[port].OUTCLR.reg = pinMask ;
-    break ;
+      PORT->Group[port].OUTCLR.reg = pinMask;
+    break;
 
     case OUTPUT:
       // enable input, to support reading back values, with pullups disabled
-      PORT->Group[port].PINCFG[pin].reg=(uint8_t)(PORT_PINCFG_INEN) ;
+      PORT->Group[port].PINCFG[pin].reg = (uint8_t) (PORT_PINCFG_INEN | PORT_PINCFG_DRVSTR);
```
Comment (hathach) [inline cores/arduino/wiring_digital.c:66]: PORT_PINCFG_DRVSTR for output pin, the rest is formatting.

Comment (WestfW) [reply cores/arduino/wiring_digital.c:66]: Looks good to me (which is to say it's exactly the patch I imagined, if I were brave enough to make this change across all SAMD platforms.)

**Review by ladyada — APPROVED** [review]

**Comment by ladyada** [comment]
@hathach thanks :) can do a release now - i also added the feather m4 adalogger so a release is due

**Comment by hathach** [comment]
> @hathach thanks :) can do a release now - i also added the feather m4 adalogger so a release is due

will do the release asap

**Pull request merged**